rollup.config.js

```diff
@@ -70,6 +77,9 @@ export default name => [
           },
         },
       }),
+      invariantPlugin({
+        errorCodes: true,
```

Comment:
See https://github.com/apollographql/apollo-client/pull/4521 for more explanation of what this option does.

Reply:
That's actually a really nice solution for having some errors in production. That rollup plugin is really smart